pkg/runctl/cfg/config.go

```diff
@@ -33,62 +33,83 @@ const (
 // PipelineRunsConfigStruct is a struct holding the pipeline runs configuration
 type PipelineRunsConfigStruct struct {
 	Timeout                                       *metav1.Duration
-	NetworkPolicies                               map[string]string
-	DefaultNetworkPolicy                          string
 	LimitRange                                    string
 	ResourceQuota                                 string
 	JenkinsfileRunnerImage                        string
 	JenkinsfileRunnerImagePullPolicy              string
 	JenkinsfileRunnerPodSecurityContextRunAsUser  *int64
 	JenkinsfileRunnerPodSecurityContextRunAsGroup *int64
 	JenkinsfileRunnerPodSecurityContextFSGroup    *int64
+
+	NetworkPolicies      map[string]string
+	DefaultNetworkPolicy string
 }
 
 // LoadPipelineRunsConfig loads the pipelineruns configuration and returns it.
 func LoadPipelineRunsConfig(clientFactory k8s.ClientFactory) (*PipelineRunsConfigStruct, error) {
 	configMapIfce := clientFactory.CoreV1().ConfigMaps(system.Namespace())
-	config := &PipelineRunsConfigStruct{}
+	dest := &PipelineRunsConfigStruct{}
+
+	processConfigMap := func(
```

Comment:
May be make this a top level function for simpler testing?

Reply:
See 2aa3200.